mobile/src/Student/components/Answers/Answer/index.js

```diff
@@ -1,129 +1,200 @@
+          <LinearGradient
+            colors={["#98CA28", "#49B036"]}
+            style={styles.headerContainer}
+            start={{ x: 0, y: 0 }}
+            end={{ x: 1, y: 1 }}
+          >
+            <Image
+              source={require("../img/Team1Pick.png")}
+              resizeMethod="resize"
+              resizeMode="contain"
+              width={scale(45)}
+              style={styles.image}
+            />
+            {showPercentage && (
+              <Animated.Text
+                style={[
+                  styles.totalPercentage,
+                  {
+                    color: "white",
+                    position: "absolute",
+                    bottom: 0,
+                    right: 0,
+                    backgroundColor: "#00000000",
+                    opacity: progressBarFade,
+                  },
+                ]}
+              >
+                {percentageTitle(percentage)}
+              </Animated.Text>
+            )}
+          </LinearGradient>
+        </Animated.View>
+      )}
+      {percentageVisible && showPercentage && (
+        <Animated.Text
+          style={[
+            styles.totalPercentage,
             {
-                percentageVisible && <Text
-                    style={[
-                        styles.totalPercentage,
-                        {
-                            color: isAnswer ? 'white' : '#C7C7C7',
-                            backgroundColor: isAnswer ? '#00000000' : '#F7F6F6',
-                        }
-                    ]}
-                >
-                    {percentageTitle(percentage)}
-                </Text>
-            }
+              color: showAnswer ? "white" : "#C7C7C7",
+              backgroundColor: showAnswer ? "#00000000" : "#F7F6F6",
+              opacity: progressBarFade,
+            },
+          ]}
+        >
+          {percentageTitle(percentage)}
+        </Animated.Text>
+      )}
+    </Animated.View>
+  );
+};
 
-        </View>
-    )
-}
-
-export default Answer
+export default Answer;
 
 const styles = StyleSheet.create({
-    container: {
-        flex: 1,
-        flexDirection: 'row',
-        borderColor: '#E4E4E6',
-        borderRadius: 12,
-        borderWidth: 1,
-        height: 50,
-        overflow: 'hidden'
-    },
-    answer: {
-        fontSize: fonts.xMedium,
-        fontFamily: fontFamilies.karlaRegular,
-        fontWeight: 'bold',
-        paddingLeft: 9,
-        textAlignVertical: 'center',
-        flex: 1,
-    },
-    totalPercentage: {
-        width: scale(45),
-        textAlignVertical: 'center',
-        textAlign: 'center',
-        fontFamily: fontFamilies.karlaRegular,
-        fontSize: fonts.xMedium,
-        fontWeight: '700'
-    },
-    image: {
-
-    },
-    answerContainer: {
-        flex: 1,
-        flexDirection: 'column',
-    },
-    progressBar: {
-        marginLeft: scale(8),
-        marginBottom: scale(4),
-    }
-})
+  container: {
```

Comment:
Looks like your tabs are at 2space so it shows all the lines as changed instead of the newly added ones

Reply:
Yup I have made it to be 4 now